modules/percolator/src/test/java/org/elasticsearch/percolator/PercolatorFieldMapperTests.java

```diff
@@ -413,7 +412,7 @@ public void testNestedPercolatorField() throws Exception {
                             .field("query_field", queryBuilder)
                         .endObject().endObject().bytes(),
                         XContentType.JSON));
-        assertThat(doc.rootDoc().getFields().size(), equalTo(11)); // also includes all other meta fields
+        assertThat(doc.rootDoc().getFields().size(), equalTo(9)); // also includes all other meta fields
```

Comment:
That is because with a single type, the `_type` indexed and stored fields are left out.